generator/internal/format_class_comments.h

```diff
@@ -22,12 +22,20 @@ namespace google {
 namespace cloud {
 namespace generator_internal {
 
-/// The function does not just use service.name() and takes @p service_name in
-/// case there exists a service_name_mapping argument that is renaming the
-/// service.
+/* The function formats class comments based on the pre-existing service
```

Comment:
Restore doxygen-ness?

Reply:
I think adding the ** suffices.

Or do you mean make the whole thing a comment with ///

Reply:
Either way.